clients/include/bytes.hpp

```diff
@@ -222,6 +236,13 @@ constexpr double her2_gbyte_count(int n)
     return (sizeof(T) * (tri_count(n) + 2 * n)) / 1e9;
 }
 
+/* \brief byte  counts of SYR */
+template <typename T>
+constexpr double syr_gbyte_count(int n)
+{
+    return (sizeof(T) * (tri_count(n) * 2)) / 1e9;
```

Comment:
This is same as rocBLAS but wasn't sure if the reads for x and y should be  + 2 * n .

Reply:
Looks like we've had a similar conversation before :) https://github.com/ROCmSoftwarePlatform/rocBLAS-internal/pull/440 and [LWPMLSE-88](http://ontrack-internal.amd.com/browse/LWPMLSE-88)

I've updated syr, syr2, spr and spr2 for now. I'll open another PR in rocBLAS with the same changes before merging this for some more exposure. Let me know if they look ok to you.